src/test/java/io/reactivex/flowable/FlowableNullTests.java

```diff
@@ -2251,6 +2251,16 @@ public void throttleLastSchedulerNull() {
         just1.throttleLast(1, TimeUnit.SECONDS, null);
     }
 
+    @Test(expected = NullPointerException.class)
```

Comment:
This is not necessary, null checks are done automatically in `ParamValidationCheckerTest`.